ros/src/computing/perception/prediction/motion_predictor/packages/naive_motion_predict/nodes/naive_motion_predict/naive_motion_predict.cpp

```diff
@@ -215,8 +215,6 @@ bool NaiveMotionPredict::isObjectValid(const autoware_msgs::DetectedObject &in_o
       std::isnan(in_object.pose.position.x) ||
       std::isnan(in_object.pose.position.y) ||
       std::isnan(in_object.pose.position.z) ||
-      (in_object.pose.position.x <= 0) ||
```

Comment:
This would stop predicting objects behind and to the right (in lidar frame). is this what you intended? 
or you meant `fabs(x) <0` ??

Reply:
![glossary_scheme_128](https://user-images.githubusercontent.com/10920881/54326781-ff85e500-464a-11e9-919d-e6535ff50653.gif)

Currently only predicting objects in the above image's area.

>is this what you intended?

I think `prediction` needs to predicts objects without constraining objects' location.

Do you have a reason for restricting objects' location?

Reply:
We would need to add a threshold to stop predicting points hitting the ego-vehicle chassis. 
`fabs(x) < THRESHOLD` and `fabs(y) < THRESHOLD` would work for this purpose.